_events/2019-04-02-nescala.md

```diff
@@ -0,0 +1,10 @@
+---
+category: event
+title: Northeast Scala Symposium
+logo: /resources/img/nescala.png
+location: Philadelphia, PA
+description: "The Scala conference of the Northeastern US"
+start: 2 April 2019
+end: 3 April 2019
+link-out: http://www.nescala.org/
```

Comment:
it's not yet on the website, but announced here: https://twitter.com/nescalas/status/1086306457556054016